src/containers/guest-home-page/reset-password/ResetPassword.tsx

```diff
@@ -76,13 +82,13 @@ const ResetPassword = ({ resetToken, openModal }) => {
     }
   }, [error, openModal, response, dispatch, successNotification])
 
-  const { handleSubmit, handleInputChange, handleBlur, errors, data } = useForm(
-    {
-      onSubmit: () => sendResetPassword({ password: data.password }),
+  const { handleSubmit, handleInputChange, handleBlur, errors, data } =
+    useForm<NewPassword>({
+      onSubmit: (): Promise<void> =>
```

Comment:
Maybe it is better to write it like `onSubmit: async () =>` because in that way it would be clearer for other developers that this is an asynchronous function

Reply:
fixed